common/Perf/Azure.Test.Perf/Azure.Test.Perf.csproj

```diff
@@ -4,10 +4,9 @@
       Perf framework should depend on lowest version of Azure.Core for maximum compat, allowing test
       projects to choose a higher version if desired.
     -->
-    <PackageReference Include="Azure.Core" VersionOverride="1.0.0" />
+    <PackageReference Include="Azure.Core" VersionOverride="1.40.0" />
```

Comment:
1.40.0 selected because it ensures that the dependency tree aligns with more recent versions, and is still more than 18 months old, so anything being tested is expected to be newer.